great-idea/css/index2.css

```diff
@@ -0,0 +1,94 @@
+body {
+    margin: 60px 100px;
+  }
+  
+  header nav a {
+    color: gray;
+    margin: 35px 3%;
+    text-decoration: none;
+  }
+  
+  header nav {
+    width: 80%;
+  }
+  
+  header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 30px
+  }
+  
+  .services-header-img {
+    width: 100%;
+    margin-bottom: 35px;
+  }
+  
+  h1 {
+    font-size: 30px;
+  }
+  
+  article p {
+    margin-bottom: 35px;
+  }
+  
+  .middle-content {
+    border-style: solid none solid none;
+    border-width: 1px;
+    padding: 40px 0;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
+  }
+  
+  .info-cards {
+    border: 1px solid;
+    width: 500px;
+    height: 220px;
+    margin: 20px 0;
+    background-color: #eee;
+    padding: 20px;
+  }
+  
+  button {
+    margin-top: 20px;
+    background-color: white;
+    width: 170px;
+    height: 40px;
+    border: 1px solid;
+    border-radius: 5px;
+  }
+  
+  button:hover {
+    background-color: black;
+    color: white;
+  }
+  
+  .bottom-content {
+    margin: 40px 0;
```

Comment:
Great job using the TRBL model to provide multiple arguments here, using the 2 argument syntax (vs 1, 3, or 4 
|or| using multiple margin property like `margin-top` and `margin-bottom`)